doctest/build.gradle

```diff
@@ -3,7 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+
 import java.util.concurrent.Callable
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
```

Comment:
`internal`? There are a several ways to check current platform in Gradle. Why did you choose this one?

Reply:
I tried few of them and found this one working without any adding a dependency.